.github/workflows/build.yaml

```
@@ -0,0 +1,170 @@
+          java-version: 21
+          cache: maven
+
+      - name: Build & run tests
+        run: mvn clean verify
+
+  # ────────────────────────────────────────────────────────────────
+  # Package & Upload to S3 on main or manual dispatch
+  # ────────────────────────────────────────────────────────────────
+  package:
+    needs: build-test
+    if: >
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)